src/main/kotlin/no/nav/syfo/client/HttpClientCommon.kt

```diff
@@ -15,3 +18,16 @@ fun httpClientDefault() = HttpClient(CIO) {
         connectTimeoutMillis = 60000
     }
 }
+
+val proxyConfig: HttpClientConfig<ApacheEngineConfig>.() -> Unit = {
+    install(JsonFeature) {
+        serializer = JacksonSerializer(configuredJacksonMapper())
+    }
+    engine {
+        customizeClient {
+            setRoutePlanner(SystemDefaultRoutePlanner(ProxySelector.getDefault()))
+        }
+    }
+}
+
+fun httpClientProxy() = HttpClient(Apache, proxyConfig)
```

Comment:
Kunne httpClientApache være et bedre navn (dvs at det er et alternativ til httpClientDefault). 

@janzh Vet du forresten hvorfor CIO-klienten ikke kan brukes i denne sammenhengen? (mener vi hadde en runde på det for en stund siden, men skjønte ikke helt hvorfor det virker bedre med Apache)

Reply:
Mener du alternativt navn til httpClientProxy?
Greia her at denne klienten har proxy i motsetning til Default som ikke har det, så da synes jeg Proxy beskriver det bedre enn Apache.

CIO-klient kan sikkert brukes med lignende proxy, men da må noen evt. lese seg opp på det.